stack-api/.gitignore

```diff
@@ -0,0 +1,20 @@
+*~
+*#
+.#*
+\#*#
+.*.sw[a-z]
+*.un~
+pkg/
+
+# Berkshelf
+.vagrant
+/cookbooks
+Berksfile.lock
```

Comment:
This shouldn't be "ignored".

Think of it as a `composer.lock` or `Gemfile.lock`, etc..